modules/ROOT/partials/configuration/spellchecker_ignore_list.adoc

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
This needs an id added to the option (all options should always have a fixed id).

Reply:
Done